packages/daemon/src/utils/helpers.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { USE_SSL, FULLNODE_HOST } from '../config';
+
+export const getFullnodeHttpUrl = () => {
+  const protocol = USE_SSL ? 'https://' : 'http://';
+
+  return `${protocol}${FULLNODE_HOST}/v1a/`;
```

Comment:
If the wallet-service runs on nodejs we can use `path.join` to safely join the url parts.

Reply:
Updated it to use URL in dfe48c88978f587fad943ad167571f7a438e694f